README.md

```diff
@@ -4,7 +4,7 @@
 # Blog-App
 
```

Comment:
- [ ] **[OPTIONAL]** I recommend you put the test files in a different folder like `spec/features`, instead of leaving them in the `spec` folder.

- [ ] Kindly organize your files. Separate the tests for the User Show page and the User Index page into different files.

- [ ] Your claim that all your tests pass locally seems not so. I have compiled the assets pipeline for JS. Most of your tests appear to be failing. Kindly fix them. In your `users_spec.rb` and `posts_spec.rb` files I would recommend you change this `type: :system` to this `type: :feature` as the former can prove problematic, after that fix the failing tests.

  ![4849](https://user-images.githubusercontent.com/36542986/200305982-0ca18e67-617c-4984-afb9-b7b9e8d26f33.jpg)

- [ ] For the User show actions you are missing the following assertions:

  - Tests that asserts you can see the user's first 3 posts.
  - Tests that asserts you can see a button that lets me view all of a user's posts.
  - Tests that redirects to the post's show page when you click on a user's post.
  - Tests that redirects to the user's post's index page when you click on a see all posts.

Reply:
@TedLivist I am not making a claim. I am sayinh when you run "bundle exec spec/users_spec.rb", the tests will run locally. You can have a zoom call with me if you have the chance to see for yourself.  If you are saying that the test coverage is incomplete is a different thing entirely. I will carefully go through them as requested and do the needful bro. Since you want an exhaustive tests. 

@TedLivist Please kindly download the files to your local machine. navigate to the appropriate branch 'testing-blog_ror_capybara' 

bundle exec spec/users_spec.rb
bundle exec spec/posts_spec.rb 

As long as we are both using same version of ruby  abd rails, same Gemfile. same rails_helper..rb setting. Then all the tests should pass. 

I checked locally and it is working on my machine. Could you please have a a zoom call with me. I will appreciate that from you. That's the essence of code review  and mentorship so as to verify my claim.

Reply:
@TedLivist Does the Google chrome pop up in your environment to display the pages of the blog app. If it does not all examples will fail

Reply:
@ogagaoghene
- we can hop on a call by 3:30. I'll reclone the repo and run the tests again
- the chrome windows open

Reply:
@TedLivist I am available bro. Lets me.